components/eventing-controller/controllers/subscription/beb/reconciler.go

```diff
@@ -633,7 +631,7 @@ func getSvcNsAndName(url string) (string, string, error) {
 // syncInitialStatus determines the desired initial status and updates it accordingly (if conditions changed)
 func (r *Reconciler) syncInitialStatus(subscription *eventingv1alpha1.Subscription) {
 	expectedStatus := eventingv1alpha1.SubscriptionStatus{}
-	expectedStatus.InitializeConditions()
+	expectedStatus.InitializeSubscriptionConditions()
```

Comment:
(nit) Why do we need to rename it? We are doing `Subscription.status.InitialiseConditions()` so according to GO recommentations we dont need to add `Subscription` word to `InitializeSubscriptionConditions`

Reply:
Its was done to differentiate between initializing `Subscription` and `Backend` conditions. But I see your point. I will change it. 👍